auth-web/src/models/continuation-review.ts

```diff
@@ -18,7 +19,7 @@ export interface ContinuationReviewResultIF {
 
 /** The Continuation Review API response. */
 export interface ContinuationReviewIF {
-  completingParty: string
+  officeEmail: string
   creationDate: string // 'YYYY-MM-DDTHH:MM.SS.000+00:00'
```

Comment:
Per Design, replace completing party with Registered Office email address:
![image](https://github.com/user-attachments/assets/4a58a71b-71a2-4a6a-8967-a97d138a9070)
Back end doesn't support this property yet